packages/react-api/__tests__/hooks/useCartoLayerFilterProps.test.js

```diff
@@ -1,15 +1,16 @@
 import { DataFilterExtension } from '@deck.gl/extensions';
 import { renderHook } from '@testing-library/react-hooks';
 import useCartoLayerProps from '../../src/hooks/useCartoLayerProps';
-import { mockClear, mockReduxHooks } from '../mockReduxHooks';
+import { mockReduxHooks, mockClear } from '../mockReduxHooks';
```

Comment:
why this order change, automatic?

Reply:
Make more sense right? First, initialization, then clearing.

Reply:
Ok, I tend to use alphabetic ordering, but it's ok. We should see if we can enforce a criteria in the future with linter